src/app/layout.tsx

```diff
@@ -7,21 +11,28 @@ import '@mantine/dropzone/styles.layer.css'
 
 import { Providers } from './providers'
 import { AppLayout } from '@/components/layout/app-layout'
-import { ReactNode } from 'react'
+import { type ReactNode } from 'react'
```

Comment:
curious why this had to become a type? i think we're doing just `ReactNode` in other components, just wondering if theres something i can learn here 😆

Reply:
heh, TBH, I don't even recall why I switched that.  As far as I know exporting a type is a little better because it makes the code easier to treeshake, the compiler will now know that it's completely safe to remove that from the build.  Of course we'll always have React included in the build so this doesn't really help at all